support/machine.go

```diff
@@ -30,3 +30,15 @@ func GetMachines(t Test, machineSetName string) []machinev1beta1.Machine {
 func MachineSetId(machineSet machinev1beta1.MachineSet) string {
 	return machineSet.Name
 }
+
+func MachineSet(test Test, namespace string, machineSetName string) func(g gomega.Gomega) *machinev1beta1.MachineSet {
+	return func(g gomega.Gomega) *machinev1beta1.MachineSet {
+		machineset, err := test.Client().Machine().MachineV1beta1().MachineSets(namespace).Get(test.Ctx(), machineSetName, metav1.GetOptions{})
+		test.Expect(err).ToNot(gomega.HaveOccurred())
```

Comment:
An alternative here to match other implementations in the file also
```suggestion
		g.Expect(err).NotTo(gomega.HaveOccurred())
```